client/js/components/button/BackToTopButton.vue

```diff
@@ -7,6 +7,7 @@
     rounded
     :style="buttonPosition"
     :text="Translator.trans('back.to.top')"
+    data-cy="DpBackToTop"
```

Comment:
I'd suggest to remove the `Dp` prefix here. Also, in other places we use lowerCamelCase.

```suggestion
    data-cy="backToTop"
```